README.md

```diff
@@ -252,21 +264,17 @@ these are configured in your Heroku app:
 - `GITLAB_ACCESS_TOKEN`
 - `GITHUB_ACCESS_TOKEN`
 - `GITHUB_WEBHOOK_SECRET`
-- `BOT_NAME` (defaults to `coqbot`)
-- `BOT_EMAIL` (defaults to `BOT_NAME@users.noreply.github.com`)
-
-In the next release of coqbot, the `BOT_NAME` and `BOT_EMAIL`
-environment variables won't have any effect and should be set from
-a configuration file instead (see [`example-config.toml`](example-config.toml)).
-The port number must not be set in the configuration file if you're
-deploying the docker image to Heroku, since the latter uses a custom
-environment variable.
+- `GITHUB_PRIVATE_KEY` (a private key of your GitHub app)
+
+Then, you must configure the bot with a configuration file. Here is an example
```

Comment:
Must or can?

Reply:
Must. Because of this [line](https://github.com/coq/bot/blob/master/src/bot.ml#L11). We can make it optional, if you prefer.